templates/vue/src/components/NodeModal.vue

```diff
@@ -257,6 +268,9 @@ export default {
         ? true
         : wpData.wpCanEditTapestry !== ""
     },
+    showMap() {
+      return this.settings.renderMap
+    },
```

Comment:
I think it would be better if you put `settings.renderMap` directly in your `v-if`. No reason to make the reader go searching for where `showMap` comes from.